engine/itest/src/test/java/org/opennms/oce/engine/itest/TopologyEngineTest.java

```diff
@@ -95,7 +95,7 @@ public void canTriggerIncidentOnCardDown() {
         assertThat(incidents, hasSize(4));
         // The 2nd incident is the Card Down and must contain the 2 alarms
         Incident incident = incidents.get(1);
-        assertThat(Level2EngineComplianceTest.getAlarmIdsInIncident(incident), containsInAnyOrder("a1", "a2"));
+        MatcherAssert.assertThat(Level2EngineComplianceTest.getAlarmIdsInIncident(incident), containsInAnyOrder("a1", "a2"));
```

Comment:
Any reason why we qualify this one particular assert with `MatcherAssert`?

Reply:
This shouldn't have been pulled in here....
I will make sure it's clean when I create my PR